pyHS100/pyHS100.py

```diff
@@ -102,3 +92,197 @@ def hs100_status(self):
+            Dict with the current readings
+        """
+        if self.model == 100:
+            return False
+
+        response = self._send_command('{"emeter":{"get_realtime":{}}}')
+
+        if response["emeter"]["get_realtime"]["err_code"] != 0:
+            return False
+
+        response["emeter"]["get_realtime"].pop('err_code', None)
+        return response["emeter"]["get_realtime"]
+
+    def get_emeter_daily(self, year=datetime.datetime.now().year, month=datetime.datetime.now().month):
+        """Gets daily statistics for a given month.
+
+          Arguments:
+            year (optional): The year for which to retrieve statistics, defaults to current year
+            month (optional): The mont for which to retrieve statistics, defaults to current month
+
+          Return values:
+            False if command is not successful or the switch doesn't support energy metering
+            Dict where the keys represent the days, and the values are the aggregated statistics
+        """
+        if self.model == 100:
+            return False
+
+        response = self._send_command('{"emeter":{"get_daystat":{"month":' + str(month) + ',"year":' + str(year) + '}}}')
+
+        if response["emeter"]["get_daystat"]["err_code"] != 0:
+            return False
+
+        data = dict()
+
+        for i, j in enumerate(response["emeter"]["get_daystat"]["day_list"]):
+            if j["energy"] > 0:
+                data[j["day"]] = j["energy"]
+
+        return data
+
+    def get_emeter_monthly(self, year=datetime.datetime.now().year):
+        """Gets monthly statistics for a given year.
+
+        Arguments:
+          year (optional): The year for which to retrieve statistics, defaults to current year
+
+        Return values:
+          False if command is not successful or the switch doesn't support energy metering
+          Dict - the keys represent the months, the values are the aggregated statistics
+        """
+        if self.model == 100:
+            return False
+
+        response = self._send_command('{"emeter":{"get_monthstat":{"year":' + str(year) + '}}}')
+
+        if response["emeter"]["get_monthstat"]["err_code"] != 0:
+            return False
+
+        data = dict()
+
+        for i, j in enumerate(response["emeter"]["get_monthstat"]["month_list"]):
+            if j["energy"] > 0:
+                data[j["month"]] = j["energy"]
+
+        return data
+
+    def erase_emeter_stats(self):
+        """Erases all statistics.
+
+          Return values:
+            True: Success
+            False: Failure or not supported by switch
+        """
+
+        if self.model == 100:
+            return False
+
+        response = self._send_command('{"emeter":{"erase_emeter_stat":null}}')
+
+        if response["emeter"]["erase_emeter_stat"]["err_code"] != 0:
+            return False
+        else:
+            return True
+
+    def current_consumption(self):
+        """Get the current power consumption in Watt."""
+
```

Comment:
Remove blankline, and also to check if smartplug supports function
<code>
        if self.model == 100:
            return False
</code>